client/src/app/components/Registerform.js

```diff
@@ -72,7 +73,7 @@ onBlur(e){
 onSubmit=(e) => {
     e.preventDefault();
     if(this.setState.length === 0){
-      console.log(errors)
+      <Redirect to='/' push/>
```

Comment:
This is better done using **React Router** and **Redux**

Reply:
agreed i have implemented a push.history